src/scaffoldmaker/utils/tubenetworkmesh.py

```diff
@@ -1062,6 +1404,18 @@ def generateMesh(self, generateData: TubeNetworkMeshGenerateData, n2Only=None):
         """
         :param n2Only: If set, create nodes only for that single n2 index along. Must be >= 0!
         """
+        # if (not n2Only) and generateData.isShowTrimSurfaces():
+        #     dimension = generateData.getMeshDimension()
+        #     nodeIdentifier, elementIdentifier = generateData.getNodeElementIdentifiers()
+        #     faceIdentifier = elementIdentifier if (dimension == 2) else None
+        #     annotationGroup = generateData.getNewTrimAnnotationGroup()
```

Comment:
Are these commented codes still needed?

Reply:
Added comment that I want to keep these for future diagnostics.